src/main/resources/messages/messages.properties

```diff
@@ -246,6 +246,12 @@ crowdin.branch.delete.usage.description=Delete a branch
 crowdin.branch.delete.usage.customSynopsis=@|fg(green) crowdin branch delete|@ <name> [CONFIG OPTIONS] [OPTIONS]
 crowdin.branch.delete.name=Branch name
 
+# CROWDIN BRANCH CLONE COMMAND
+crowdin.branch.clone.usage.description=Clone branch
+crowdin.branch.clone.usage.customSynopsis=@|fg(green) crowdin branch clone|@ <original-branch> <new-branch> [CONFIG OPTIONS] [OPTIONS]
```

Comment:
The synopsis does not match the argument names

<img width="769" alt="Screenshot 2024-04-26 at 17 24 07" src="https://github.com/crowdin/crowdin-cli/assets/29282228/61bd42ec-3454-4f8c-a61f-c7f86cd6149f">

Reply:
Lets rename the `original-branch` -> `source`, `new-branch` -> `target`